### PR TITLE
HP-5 Set up CI

### PR DESCRIPTION
**Pull request by vkurup**
I started out with CircleCI, but am looking into Github Actions since CircleCI's integration instructions have left me stuck: https://github.com/CircleCI-Public/slack-orb/issues/230

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        run: |
+          python -m pip install --upgrade pip wheel pip-tools
+          pip-sync requirements/base/base.txt requirements/dev/dev.txt
+      - name: Login to Docker
+        run: |
+          inv aws.docker-login
+      - name: Build tag, push, and deploy image
+        run: |
+          [ "$GITHUB_REF" = refs/heads/main ] &&
+            ENVIRONMENT="production" ||
```
Comment (dchukhin) [inline .github/workflows/deploy.yml:38]: Does this mean that merging into the `main` branch would deploy the `main` branch to staging and production?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+name: Lint and Test code
+
+on:
+  pull_request:
+  schedule:
+    # run once a week on early monday mornings
```
Comment (dchukhin) [inline .github/workflows/test.yml:6]: Is this in 02:22 UTC?

**Review by dchukhin — APPROVED** [review]
It looks like it works; good job with it!
I left a couple of questions, but don't have any concerns about it

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (dchukhin) [reply .github/workflows/deploy.yml:38]: Question has been answered ✔️

Comment (vkurup) [reply .github/workflows/deploy.yml:38]: Hopefully (if my googling is correct), this is a ternary statement that will set ENVIRONMENT to "production" if we're on the main branch, else to "staging", which will then be inserted into the `inv` command below to deploy the proper environment

File: .github/workflows/test.yml (hunk 2 above)
Comment (vkurup) [reply .github/workflows/test.yml:6]: Great question! we'll find out on Sunday :)

**Pull request merged**